hardhat.config.ts

```diff
@@ -63,7 +63,7 @@ const config: HardhatUserConfig = {
   paths: {
     starknetSources: './starknet',
     sources: 'ethereum/src/',
-    tests: './starknet/test',
+    tests: './starknet/tests',
```

Comment:
I just don't understand why these are in `starknet/tests` instead of `tests` ?

Reply:
ah yeah should be `tests` doesnt look like thats needed though as they ran fine

Reply:
updated